.changeset/honky-dory-fox.md

```diff
@@ -0,0 +1,12 @@
+#changelog
+---
+"app-sdk": minor
+---
+### Added
+
+- `src/APL/index.ts`: added redis-apl export
+- `src/APL/redis-apl.ts`: created Redis APL
+- `src/APL/redis-apl.test.ts`: Tests for Redis APL
+
+### Changed
+- npm run lint changed a lot of files, guess it wasn't ran in a while :)
```

Comment:
Thanks for a notice, but its not a changelog message and I don't see any linter changes committed, so we can remove this message